sequential/bank_accounts/src/bank_account.erl

```diff
@@ -2,5 +2,16 @@
 
 -export([process_operation/2]).
 
-process_operation(Bank, Operation) ->
-  put_your_solution_here.
+process_operation(Bank,
+		  {AccNumber, withdraw, Amount}) ->
```

Comment:
this indentation looks kind of strange

Reply:
My editor's formatter is killing me. I'll see to it.